app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryUpdate.kt

```diff
@@ -21,7 +21,7 @@ class CustomDrugEntryUpdate : Update<CustomDrugEntryModel, CustomDrugEntryEvent,
       is EditFrequencyClicked -> dispatch(ShowEditFrequencyDialog(event.frequency))
       is FrequencyEdited -> next(model.frequencyEdited(event.frequency), SetDrugFrequency(event.frequency), SetSheetTitle(model.drugName, model.dosage, event.frequency))
       is AddMedicineButtonClicked -> createOrUpdatePrescriptionEntry(model, event.patientUuid)
-      is CustomDrugSaved, ExistingDrugRemoved -> dispatch(CloseBottomSheet)
+      is CustomDrugSaved, ExistingDrugRemoved -> dispatch(CloseBottomSheet, GoToEditMedicineScreen)
```

Comment:
I don't think this would work, here we are closing and trying to launch the edit medicines screen. Here you're trying to do two navigation actions, I don't think this would work the same way you expect it to.

Reply:
So can I just do a `popUntil` here instead?

Reply:
Yep